[2025-05-24T12:19:06.426+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: marketing_data_pipeline.validate_website_activity manual__2025-05-24T12:18:08.583052+00:00 [queued]>
[2025-05-24T12:19:06.932+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: marketing_data_pipeline.validate_website_activity manual__2025-05-24T12:18:08.583052+00:00 [queued]>
[2025-05-24T12:19:07.233+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-24T12:19:07.486+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_website_activity> on 2025-05-24 12:18:08.583052+00:00
[2025-05-24T12:19:07.556+0000] {standard_task_runner.py:60} INFO - Started process 419 to run task
[2025-05-24T12:19:07.571+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'marketing_data_pipeline', 'validate_website_activity', 'manual__2025-05-24T12:18:08.583052+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline_dag.py', '--cfg-path', '/tmp/tmpmrs7j5wx']
[2025-05-24T12:19:07.589+0000] {standard_task_runner.py:88} INFO - Job 19: Subtask validate_website_activity
[2025-05-24T12:19:09.126+0000] {task_command.py:423} INFO - Running <TaskInstance: marketing_data_pipeline.validate_website_activity manual__2025-05-24T12:18:08.583052+00:00 [running]> on host 00e321f10423
[2025-05-24T12:19:10.068+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='yashbhawsar872@gmail.com' AIRFLOW_CTX_DAG_OWNER='Yash' AIRFLOW_CTX_DAG_ID='marketing_data_pipeline' AIRFLOW_CTX_TASK_ID='validate_website_activity' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T12:18:08.583052+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T12:18:08.583052+00:00'
[2025-05-24T12:19:10.149+0000] {main_pipeline_dag.py:25} INFO - ▶️ Starting task: Validate Website Activity
[2025-05-24T12:19:10.516+0000] {data_validation.py:207} INFO - ⏩ Already validated: website_activity_2025-05-24.json
[2025-05-24T12:19:10.526+0000] {utils.py:52} INFO - task_metrics
[2025-05-24T12:19:10.542+0000] {main_pipeline_dag.py:27} INFO - ✅ Completed task: Validate Website Activity
[2025-05-24T12:19:10.568+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-24T12:19:10.747+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=marketing_data_pipeline, task_id=validate_website_activity, execution_date=20250524T121808, start_date=20250524T121906, end_date=20250524T121910
[2025-05-24T12:19:10.962+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-24T12:19:11.189+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
